t/plugin/cors.t

```diff
@@ -927,3 +927,70 @@ Access-Control-Max-Age:
 Access-Control-Allow-Credentials:
 --- no_error_log
 [error]
+
+
+
+=== TEST 34: origin was modified by the proxy_rewrite plugin
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                 ngx.HTTP_PUT,
+                 [[{
+                    "plugins": {
+                        "cors": {
+                            "allow_origins": "http://sub.domain.com",
+                            "allow_methods": "GET,POST",
+                            "allow_headers": "headr1,headr2",
+                            "expose_headers": "ex-headr1,ex-headr2",
+                            "max_age": 50,
+                            "allow_credential": true
+                        },
+                        "proxy-rewrite": {
+                            "Origin": "http://example.com"
+                        }
```

Comment:
```suggestion
                        "proxy-rewrite": {
                            "headers": {
                                 "Origin": "http://example.com"
                            }
                        }
```

Modifying the header in proxy-rewrite should be configured like this?

Reply:
oh,  I will fix it